pyproject.toml

```diff
@@ -0,0 +1,99 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of base32-lib
+# Copyright (C) 2019 CERN.
+# Copyright (C) 2019 Northwestern University,
+#                    Galter Health Sciences Library & Learning Center.
+# Copyright (C) 2025 Front Matter.
+
+# base32-lib is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+[build-system]
+requires = ["setuptools>=45", "wheel", "setuptools_scm[toml]>=6.2"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "base32-lib"
+dynamic = ["version"]
+description = "Small library to generate, encode and decode random base32 strings."
+readme = { file = "README.rst", content-type = "text/x-rst" }
+requires-python = ">=3.8"
+license = { text = "MIT License" }
+authors = [{ name = "Invenio Software", email = "info@inveniosoftware.org" }]
+keywords = ["base32", "identifiers"]
+classifiers = [
+  "Intended Audience :: Developers",
+  "License :: OSI Approved :: MIT License",
+  "Operating System :: OS Independent",
+  "Programming Language :: Python",
+  "Programming Language :: Python :: 3",
+  "Programming Language :: Python :: 3.8",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
```

Comment:
since we drop in the near future python3.9 support i think you should remove the 3.8 and 3.9 line

Reply:
agree